dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -110,6 +109,38 @@ public QuayImageRegistry(final HttpClient client, final ObjectMapper objectMappe
         return tags;
     }
 
+    public Tag getImageInformationForTag(Tool tool, Tag tag) {
```

Comment:
I would probably call this something like updateTagWithImageInformation -- a get in Java means return a property, typically.